src/ui/qgstextformatwidgetbase.ui

```diff
@@ -35,10 +35,10 @@
       </size>
      </property>
      <property name="frameShape">
-      <enum>QFrame::NoFrame</enum>
+      <enum>QFrame::Shape::NoFrame</enum>
```

Comment:
These changes can break things on qt 5 builds -- can you revert them?

Reply:
Thanks for letting me know.
I just replaced the commit on the `.ui` file avoiding all those edits.